pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -205,3 +162,43 @@ func (am *autoScalerManager) gracefullyDeleteTidbCluster(deleteTc *v1alpha1.Tidb
 
 	return am.deps.Clientset.PingcapV1alpha1().TidbClusters(deleteTc.Namespace).Delete(deleteTc.Name, nil)
 }
+
+func (am *autoScalerManager) patchAutoscalingLastSyncingAnnotation(tac *v1alpha1.TidbClusterAutoScaler, timestamp time.Time) error {
+	mergePatch, err := json.Marshal(map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"annotations": map[string]interface{}{
+				label.AnnLastSyncingTimestamp: timestamp.Format(time.RFC3339),
+			},
+		},
+	})
+	if err != nil {
+		return err
+	}
+
+	_, err = am.deps.Clientset.PingcapV1alpha1().TidbClusterAutoScalers(tac.Namespace).Patch(tac.Name, types.MergePatchType, mergePatch)
+	if err != nil {
+		klog.Errorf("failed to update tac[%s/%s]'s annotation %s, err: %v", tac.Namespace, tac.Name, label.AnnLastSyncingTimestamp, err)
+	}
+
+	return err
+}
+
+func (am *autoScalerManager) patchAutoscalingGroupStatus(tac *v1alpha1.TidbClusterAutoScaler, memberType string, group string, newStatus *v1alpha1.BasicAutoScalerStatus) error {
```

Comment:
So with `newStatus *v1alpha1.BasicAutoScalerStatus`, do we need to update "TiKV map[string]TikvAutoScalerStatus `json:"tikv,omitempty"`" and that of TiDB?
Because it does not consider the case where xxxxAutoScalerStatus may add new fields later.